pkgs/by-name/hy/hyprland-monitor-attached/package.nix

```
@@ -0,0 +1,24 @@
+{ rustPlatform, fetchFromGitHub, lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hyprland-monitor-attached";
+  version = "unstable-2023-10-03";
```

```suggestion
  version = "0.1.5-unstable-2023-10-03";
```

Version is updated in [`Cargo.toml`](https://github.com/coffebar/hyprland-monitor-attached/commits/main/Cargo.toml)